### PR TITLE
Create aws-auth-guide-self-hosted.md

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/27179

## Description
Provide documentation the following deployment scenario: Cortex XSOAR server self-hosted outside the AWS environment in a remote network. When this is approved and added to the dev-hub, we need to add a comment in https://xsoar.pan.dev/docs/reference/articles/aws-integrations---authentication with a link to this article.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by glicht** [comment]
@ShirleyDenkberg @kirbles19 why not unite this with the other article about AWS auth? It seems that there is some overlap here.

**Comment by ShirleyDenkberg** [comment]
@glicht We decided to make this a different topic because it relates to a different use case.  From a user's perspective, I want to see information for my specific use case.  I do not want to sift through information that I do not need.e

**Comment by glicht** [comment]
Let's discuss this with @kirbles19

**Comment by kirbles19** [comment]
Combine content in this PR with https://xsoar.pan.dev/docs/reference/articles/aws-integrations---authentication

File: content-repo/extra-docs/articles/aws-auth-guide-self-hosted.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+### Create the IAM User  
+1. Log in to the AWS Management Console and access the IAM console: (https://console.aws.amazon.com/iam/)
+2. In the navigation pane, select **Users > Create user**.
+   - **User name**: add a username (for e.g., "xsoar.remediation" )
+   - **Access type**: select **Programmatic** access and click **Next**.
+   - **Set permissions**: no permissions are required. Click **Next**.
+   - **Tags** (optional): it is recommended to create a tag to describe the purpose (such us "Description: This user provide programmatic access to AWS - XSOAR integrations for auto-remediation").
+3. Finalize the user creation and save the Access key ID and Secret access key (you will need them when configuring the integrations in Cortex XSOAR). 
+
+### Create the IAM Roles for Untegrations (Assumed Roles)
```
[the remark this answers is not in the document]
Comment (ShirleyDenkberg) [reply content-repo/extra-docs/articles/aws-auth-guide-self-hosted.md]: Fixed typo

**Comment by ShirleyDenkberg** [comment]
I combined the content in this PR with https://xsoar.pan.dev/docs/reference/articles/aws-integrations---authentication.

**Comment by glicht** [comment]
Closing. Was replaced by: https://github.com/demisto/content-docs/pull/270

**Pull request closed without merging**